packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesController.kt

```diff
@@ -62,6 +63,14 @@ class UpdatesController private constructor(
     }
   }
 
+  private fun purgeUpdatesLogsOlderThanOneDay() {
+    UpdatesLogReader(context).purgeLogEntries {
+      if (it != null) {
+        Log.e(TAG, "UpdatesLogReader: error in purgeLogEntries", it)
+      }
+    }
```

Comment:
optional: I generally read `it` as a non-error, but that might just be me

```suggestion
    UpdatesLogReader(context).purgeLogEntries { error ->
      if (error != null) {
        Log.e(TAG, "UpdatesLogReader: error in purgeLogEntries", error)
      }
    }
```